ocs_ci/templates/ocs-deployment/provider-mode/ocs_storagecluster.yaml

```diff
@@ -4,7 +4,6 @@ metadata:
   name: ocs-storagecluster
   namespace: openshift-storage
 spec:
-  allowRemoteStorageConsumers: true
```

Comment:
Because of upgrade job reason - where we do deploy from target version currently - and what we have decided long time back when we brought up release branches is that we will support ODF N-1 for deployment and pre-upgrade tests.

So our automation is using master/stable branch to deploy also ODF 4.18 once OCS-CI will be defaulting master to 4.19.  So as I explained on meeting, this should be backward compatible - to still allow to deploy ODF 4.18.

Reply:
Thank you! 
Working on this.